stdlib/public/core/CMakeLists.txt

```diff
@@ -325,7 +325,7 @@ set(swiftCore_common_options
                   FRAMEWORK_DEPENDS
                     ${swift_core_framework_depends}
                   SWIFT_COMPILE_FLAGS
-                    ${swift_stdlib_compile_flags} -Xcc -DswiftCore_EXPORTS ${SWIFT_STANDARD_LIBRARY_SWIFT_FLAGS})
+                    ${swift_stdlib_compile_flags} -Xcc -DswiftCore_EXPORTS -enable-experimental-feature Extern ${SWIFT_STANDARD_LIBRARY_SWIFT_FLAGS})
```

Comment:
Could you put this flag underneath the other experimental flags defined above?

```cmake
list(APPEND swift_stdlib_compile_flags "-enable-experimental-feature" "Macros")
list(APPEND swift_stdlib_compile_flags "-enable-experimental-feature" "FreestandingMacros")
```

Reply:
Thanks!